students/tri_nguyen/lesson06/test_mailroom.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+
+# ------------------------------------------- #
+# Student:   Tri Nguyen
+# Instructor: Natasha Aleksandrova
+# Date:  16-Mar-2018
+# ------------------------------------------- #
+
+import unittest
+import mailroom
+
+class TestMailroom(unittest.TestCase):
+
+    def test_display_menu(self):
+        ''' test display_menu function of mailroom.py '''
+
+        known_menu = ''' Select one of the actions below:
+
+    1. Send a Thank You
+    2. Create a Report
+    3. Send letters to everyone
+    4. Quit
+
+    '''
+
+        expected_result = mailroom.display_menu()
+        self.assertEqual(expected_result, known_menu)
+        # are assertIsNotNone and assertIsInstance necessary??
```

Comment:
It's good to be thinking this way when writing tests, and checking for existence and type are useful things to test for in some cases, however since the function you're testing here is only assigning a string to a variable, I don't think you need these assertions